heapster.go

```diff
@@ -36,10 +36,12 @@ var (
 	argPollDuration    = flag.Duration("poll_duration", 10*time.Second, "The frequency at which heapster will poll for stats")
 	argStatsResolution = flag.Duration("stats_resolution", 5*time.Second, "The resolution at which heapster will retain stats. Acceptable values are in the range [1 second, 'poll_duration')")
 	argAlignStats      = flag.Bool("align_stats", false, "Whether to align timestamps of metrics to multiplicity of 'stats_resolution'")
+	argCacheDuration   = flag.Duration("cache_duration", 10*time.Minute, "The total duration of the historical data that will be cached by heapster.")
+	argUseModel        = flag.Bool("use_model", false, "When true, the internal model representation will be used.")
+	argModelResolution = flag.Duration("model_resolution", 20*time.Second, "The resolution of the timeseries stored in the model. Applies only if use_model is true.")
```

Comment:
I just noticed that the other flags don't end with a period. Perhaps do the same?